Creacion de Entities

edgaremirborbonjimenez opened the pull request.

Marcos12M committed:
Entidades

Se crearon las entidades y sus relaciones.

Co-Authored-By: Edgar Emir Borbón Jimenez <98839950+edgaremirborbonjimenez@users.noreply.github.com>

edgaremirborbonjimenez committed:
EntityModule

Hice el Modulo de Entities y agregue el Enum de Estatus del Pedido

edgaremirborbonjimenez committed:
Se cambio anotacion

Se cambio anotacion de Empleado y Tortilleria en un ManyToMany

edgaremirborbonjimenez committed:
Correccion de Entities

Corregi accesos de datos Entidades
y lo homologue al doagrama de dominio, Agregue una clase Usuario para  hacer las herencias de Repartidor y Empleado

edgaremirborbonjimenez committed:
Merge remote-tracking branch 'origin/development' into feat-entidades

edgaremirborbonjimenez committed:
ajuste

SilverFlin commented on src/entities/classes/repartidor.entity.ts:
Hay varios imports que no se usan acá, no corriste `npm run lint` (Y de paso `npm run format`)?

edgaremirborbonjimenez committed:
hice limpia entidades datos

DanielPena229185 committed:
Validaciones en atributos de entidades

Agregué nullables para ser más específico y además agregué los nombres a las columnas, además de personalizar el nombre de la tabala MM entre empleados y tortillerías

DanielPena229185 commented on src/entities/classes/pedido.entity.ts:
```diff
@@ -52,11 +51,11 @@ export class Pedido {
 
   @OneToMany(() => Paquete, (paquete) => paquete.pedido, {
-    cascade: ['remove'],
+    cascade: ['remove', 'update'],
```
Agregué el **update** porque consideré que si la tienda modifica el pedido, pueda modificar los paquetes también
